doc/audit/builtin_maps.rst

```diff
@@ -1,29 +1,264 @@
 Built-in Maps
 =============
 
-`public:ccf.gov.nodes.code_ids`
--------------------------------
+`public:ccf.gov.`
+-----------------
 
-This table contains all the versions of the code allowed to join the current network.
+`certs`
+~~~~~~~
+
+X509 certificates of all members in the consortium.
+
+**Key** Member ID: SHA-256 digest of the member certificate, represented as a hex-encoded string.
+
+**Value** Member certificate, represented as a PEM-encoded string.
+
+`encryption_public_keys`
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Public encryption keys submitted by members to the network. The recovery share for each member is encrypted by the key they have submitted.
+
+**Key** Member ID: SHA-256 digest of the member certificate, represented as a hex-encoded string.
```

Comment:
Here and elsewhere:
```suggestion
**Key** Member ID: SHA-256 digest of the DER-encoded member certificate, represented as a hex-encoded string.
```

Reply:
Yes, absolutely, very well spotted, I will fix this everywhere.

Reply:
It's also often referred to as the SHA-256 fingerprint of the certificate, which is more concise.